src/index.ts

```diff
@@ -707,7 +707,13 @@ class SpvWalletClient implements TransportService {
     draftTransaction.configuration.inputs?.forEach((input, index) => {
       if (input.destination) {
         const chainKey = xPriv.deriveChild(input.destination.chain);
```

Comment:
Make a destruction for this.

`const {num, chain, paymail_external_derivation_num = undefined} = input.destination;`

Reply:
why?